src/commands/game_commands/preset.ts

```diff
@@ -118,132 +215,202 @@ export default class PresetCommand implements BaseCommand {
         priority: 200,
     });
 
-    call = async ({ message, parsedMessage }: CommandArgs): Promise<void> => {
-        const guildPreference = await GuildPreference.getGuildPreference(
-            message.guildID
-        );
+    slashCommands = (): Array<Eris.ChatInputApplicationCommandStructure> => [
+        {
+            name: "preset",
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.preset.help.description"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: PresetAction.LIST,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.preset.help.example.list"
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .SUB_COMMAND,
+                },
+                {
+                    name: PresetAction.SAVE,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.preset.help.example.save"
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .SUB_COMMAND,
+                    options: [
+                        {
+                            name: "preset_name",
+                            description:
+                                LocalizationManager.localizer.translate(
+                                    LocaleType.EN,
+                                    "command.preset.help.example.save"
```

Comment:
May need a new translation string for the slash command options based on the context
save : the name to save the preset as
load/delete: the name of the preset to load/delete
replace: the preset to replace
import: the unique identifier associated with the preset